### PR TITLE
solution

### PR DESCRIPTION
**Pull request by DaniilBarilotti**
https://daniilbarilotti.github.io/layout_product-cards/
https://daniilbarilotti.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+        class="card__image card__image--position"
+      >
+      <h2 class="card__name card__name--position">
+        APPLE A1419 IMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__product-code card__product-code--position">
+        Product code: 195434
+      </p>
+      <div class="stars stars--position">
+        <div class="stars--4 stars">
```
Comment (VitaliyBondarenko1982) [inline src/index.html:28]:
```suggestion
        <div class="stars stars--4">
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+    rel="stylesheet"
+    >
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body class="page"> 
+    <div class="card" data-qa="card">
+      <img 
+        src="images/imac.jpeg" 
+        alt="img" 
+        class="card__image card__image--position"
```
Comment (VitaliyBondarenko1982) [inline src/index.html:19]: Do you really need modifier card__image--position?

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
+      >
+      <h2 class="card__name card__name--position">
```
Comment (VitaliyBondarenko1982) [inline src/index.html:21]: Do you really need modifier `card__name--position`?
As style.css is empty now? i can't see for what it.

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,50 @@
...
+        <div class="stars--4 stars">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <p class="stars__reviews">Reviews: 5</p>
+      </div> 
+      <div class="card__price card__price--position">
```
Comment (VitaliyBondarenko1982) [inline src/index.html:37]: Same as above comments about modifier

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Maybe you forgot to push style.css
Screenshot from 2023-03-05 16-34-01

**Comment by DaniilBarilotti** [comment]
I accidentally deleted style.css and just created a new file, but it didn't help.  I will create a new fork and put my code there.

**Pull request closed without merging**